redash/handlers/groups.py

```diff
@@ -0,0 +1,10 @@
+from redash.handlers.base import BaseResource
+from redash.wsgi import api
+from redash.models import Group
+
+
+class GroupsAPI(BaseResource):
+    def get(self):
+        return [group.to_dict() for group in Group.select()]
```

Comment:
If the user isn't an admin, it should only return the list of groups the user belongs to.